AppCheck/Sources/AppAttestProvider/GACAppAttestProvider.m

```diff
@@ -143,11 +148,17 @@ - (nullable instancetype)initWithApp:(FIRApp *)app {
   GACAppAttestKeyIDStorage *keyIDStorage =
       [[GACAppAttestKeyIDStorage alloc] initWithAppName:app.name appID:app.options.googleAppID];
 
+  GACAppCheckAPIRequestHook heartbeatLoggerHook = ^(NSMutableURLRequest *request) {
```

Comment:
This is temporary also, right?

Reply:
Yep, thanks for catching! I just added some comments around these code blocks in each of the existing providers so they don't get missed.

Note: The request hooks concept will likely remain (or some alternative approach) but I'll need to pass them in from the Firebase App Check SDK since the generic App Check SDK shouldn't know about `FIRHeartbeatLoggerProtocol` once we're done refactoring (these code blocks themselves are temporary though).